.github/workflows/benchmark.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore: ["docs/**", "**.md"]
     branches:
       - main
-  pull_request:
+  pull_request_target:
     paths-ignore: ["docs/**", "**.md"]
     types: [opened, reopened, synchronize, labeled]
```

Comment:
Do you want `edited` in case the base branch changes?

Reply:
Unsure about this one